src/sentry/search/eap/uptime_check_columns.py

```diff
@@ -0,0 +1,93 @@
+from sentry_protos.snuba.v1.request_common_pb2 import TraceItemType
+
+from sentry.search.eap import constants
+from sentry.search.eap.columns import (
+    ColumnDefinitions,
+    ResolvedColumn,
+    VirtualColumnDefinition,
+    datetime_processor,
+    project_context_constructor,
+    project_term_resolver,
+)
+from sentry.search.eap.common_columns import COMMON_COLUMNS
+
+UPTIME_CHECK_ATTRIBUTE_DEFINITIONS = {
+    column.public_alias: column
+    for column in COMMON_COLUMNS
+    + [
+        ResolvedColumn(
+            public_alias="environment",
+            internal_name="sentry.environment",
+            search_type="string",
+        ),
+        ResolvedColumn(
```

Comment:
will want to figure out how we resolve this column -- this is not optimal in its current form